sql/api/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBaseLexer.g4

```diff
@@ -217,6 +217,7 @@ HOURS: 'HOURS';
 IDENTIFIER_KW: 'IDENTIFIER';
 IF: 'IF';
 IGNORE: 'IGNORE';
+IMMEDIATE: 'IMMEDIATE';
```

Comment:
Is it reserved keyword or not. You should mention this in a doc.

Reply:
It should be reserved keyword, right ?
I should mention that in `sql-ref-syntax-aux-exec-imm.md`?

Reply:
it seems to be a reserved keyword in standard

Reply:
IMMEDIATE is not a reserved keyword, whereas EXECUTE is reserved in ANSI